src/data/finsupp/basic.lean

```diff
@@ -1591,7 +1591,7 @@ end
 
 /-- Given `add_comm_monoid β` and `e : α₁ ≃ α₂`, `dom_congr e` is the corresponding `equiv` between
 `α₁ →₀ β` and `α₂ →₀ β`. -/
-protected def dom_congr [add_comm_monoid β] (e : α₁ ≃ α₂) : (α₁ →₀ β) ≃ (α₂ →₀ β) :=
+protected def dom_congr [add_comm_monoid β] (e : α₁ ≃ α₂) : (α₁ →₀ β) ≃+ (α₂ →₀ β) :=
```

Comment:
Could you please switch this definition to an explicit `{ to_fun := _, inv_fun := _, ... }` style?